pkg/network/bridge/plugin.go

```diff
@@ -0,0 +1,218 @@
+package bridge
+
+import (
+	"container/list"
+	"fmt"
+	"math/rand"
+	"os/exec"
+	"time"
+
+	"github.com/golang/glog"
+	"github.com/vishvananda/netlink"
+	"golang.org/x/net/context"
+	pluginapi "k8s.io/kubernetes/pkg/kubelet/apis/deviceplugin/v1alpha"
+	"kubevirt.io/kubernetes-device-plugins/pkg/dockerutils"
+	"kubevirt.io/kubernetes-device-plugins/pkg/dpm"
+)
+
+const (
+	fakeDevicePath    = "/tmp/deviceplugin-network-bridge-fakedev"
+	resourceNamespace = "bridge.network.kubevirt.io/"
+)
+
+type NetworkBridgeDevicePlugin struct {
+	dpm.DevicePlugin
+	bridge       string
+	attachmentCh chan *Attachment
```

Comment:
I wonder if bridges use `attachment` terminology - at least in VFIO world, we've came to conclusion that `assignment` is tad better match.

Reply:
Done